src/Serializer/AbstractItemNormalizer.php

```diff
@@ -203,6 +203,10 @@ public function denormalize(mixed $data, string $class, string $format = null, a
 
             unset($context['input'], $context['operation'], $context['operation_name']);
             $context['resource_class'] = $inputClass;
+            $context['from_input'] = true;
+            if (false === isset($context['groups']) || [] === $context['groups']) {
+                $context['groups'][] = '*';
```

Comment:
it looks quite dangerous to me to replace groups that are not set as this is not the same behavior

Reply:
https://github.com/symfony/serializer/blob/6.3/Normalizer/AbstractNormalizer.php#L253

Only because of this check. In case we don't have groups this method will return false instead of list allowed attributes